src/main/java/org/folio/inventory/resources/Instances.java

```diff
@@ -547,14 +549,17 @@ private void successResponse(RoutingContext routingContext, WebContext context,
   private CompletableFuture<InstancesResponse> withInstancesRelationships(
     InstancesResponse instancesResponse, List<JsonObject> relationsList) {
 
+    Map<String, List<InstanceRelationshipToRelated>> relatedInstanceMap = new HashMap<>();
     Map<String, List<InstanceRelationshipToParent>> parentInstanceMap = new HashMap<>();
     Map<String, List<InstanceRelationshipToChild>> childInstanceMap = new HashMap<>();
 
     relationsList.forEach(rel -> {
+      addToList(relatedInstanceMap, rel.getString("relatedInstanceId"), new InstanceRelationshipToRelated(rel));
```

Comment:
Why are related instances being mixed together with parent and child instances?

As I understand it, these are unconnected ideas. 

I believe the reason the parent and child instances are combined together is due to how the direction of these relationships work. Meaning we need to do some translation between the business logic representation and the storage representation when figuring out changes.

How do related instances fit into this, given that they are stored separately?

Reply:
From your comments (this one and others) it is clear to me that I have misinterpreted the intent of this issue. I will be pushing up a different take on things on Monday, and hopefully, it will be more in line with the intent of the issue.

Reply:
> From your comments (this one and others) it is clear to me that I have misinterpreted the intent of this issue. 

I think I may have caused confusion by describing the requirements for this work as a comparison of instance relationship. Giving the impression that the two are variants of the same thing, rather two separate ideas that are modelled in a similar way.

I did / do find the naming of these two ideas and their modelling confusing. When we were refining this work, I asked about why they were being modelled separately in these ways and I don't fully understand the reasoning for it.

> I will be pushing up a different take on things on Monday, and hopefully, it will be more in line with the intent of the issue.

Thank you for being open to the feedback and being willing to rework this work.